lib/services/screen-builder-service.ts

```diff
@@ -131,17 +131,35 @@ export class ScreenBuilderService implements IScreenBuilderService {
 $injector.register("screenBuilderService", ScreenBuilderService);
 
 class ScreenBuilderDynamicCommand implements ICommand {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js", "tmp.app.json"];
```

Comment:
No *tmp.app.json* is present in a freshly created project - this would render newly created projects useless